### PR TITLE
[MRG] _part- -> _split- to conform with BIDS 

### PR DESCRIPTION
**Pull request by sappelhoff**
see: https://github.com/bids-standard/bids-specification/pull/460

to be merged before CIs pass:
- [x] https://github.com/mne-tools/mne-python/pull/7794
    - [x] backported to 0.20?
- [x] https://github.com/bids-standard/bids-validator/pull/960

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [x] All comments resolved
- [x] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- ~PR description includes phrase "closes <#issue-number>"~
- [x] Commit history does not contain any merge commits

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -26,6 +26,7 @@ Changelog
 - :func:`mne_bids.utils.get_entity_vals` has gained ``ignore_*`` keyword arguments to exclude specific values from the list of results, e.g. the entities for a particular subject or task, by `Richard Höchenberger`_ (`#409 <https://github.com/mne-tools/mne-bids/pull/409>`_)
 - :func:`mne_bids.write_raw_bids` now uses the 'space' BIDS-entity when writing iEEG electrodes and coordinate frames by `Adam Li`_ (`#390 <https://github.com/mne-tools/mne-bids/pull/390>`_)
 - :code:`convert_ieeg_to_bids` to now use sample ECoG EDF data by `Adam Li`_ (`#390 <https://github.com/mne-tools/mne-bids/pull/390>`_)
+- BIDS conformity: The ``_part-%d`` entity is now called ``_split-`` throughout BIDS, MNE, and MNE-BIDS, by `Stefan Appelhoff`_ (`#417 <https://github.com/mne-tools/mne-bids/pull/417>`_)
```
Comment (jasmainak) [inline doc/whats_new.rst:29]: I think this should go into the API section

**Comment by sappelhoff** [comment]
appveyor is green, so I trust that Travis will also be green soon.

This is ready to be merged and then we can close this chapter :-)

File: .travis.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ matrix:
 env:
   global:
     # can be "stable", or anything that can be used with git checkout
-    - VALIDATOR_VERSION="8887074e5f5f48a768842ffcf96fb2952dbe0e38"
+    - VALIDATOR_VERSION="e457907ca9af4c32aede828117ea5a43737b4597"
```
Comment (jasmainak) [inline .travis.yml:39]: why this random commit??! the stable branch doesn't yet work?

Comment (sappelhoff) [reply .travis.yml:39]: The bids-validator releases very often (sometimes even twice a week), so I find it more "stable" for us to regularly pin to an up to date commit.

that particular commit corresponds to my merge of the `split` rules into the validator.

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**